docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "ASAP - Testing",
+  "type": "collection"
+}
\ No newline at end of file
```

Comment:
It's a good practice to add a newline at the end of the file. This can prevent potential issues with version control systems and other tools that expect or require a newline at the end of a file. [medium]